www/source/index.html.slim

```diff
@@ -141,7 +134,7 @@ header.blue-gradient.margin-top-offset
             br/
             span.code-attribute end
 
-          a.btn.strict-right.t-white.try-demo.mobile-hide Try the Demo     
+          a.btn.strict-right.t-white.try-demo.mobile-hide href="http://inspec.io/tutorial" Try the Demo     
```

Comment:
Same comment as above - can we make this a relative URL instead of a direct URL?